frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenuCards/FeatureStrategyMenuCards.tsx

```diff
@@ -1,28 +1,89 @@
 import { useMemo } from 'react';
-import { List, ListItem } from '@mui/material';
+import { List, ListItem, styled, Typography } from '@mui/material';
 import { useStrategies } from 'hooks/api/getters/useStrategies/useStrategies';
 import { FeatureStrategyMenuCard } from '../FeatureStrategyMenuCard/FeatureStrategyMenuCard';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import useProject from 'hooks/api/getters/useProject/useProject';
+import { IStrategy } from 'interfaces/strategy';
+import useUiConfig from '../../../../../hooks/api/getters/useUiConfig/useUiConfig';
 
 interface IFeatureStrategyMenuCardsProps {
     projectId: string;
     featureId: string;
     environmentId: string;
 }
 
+const StyledTypography = styled(Typography)(({ theme }) => ({
+    fontSize: theme.fontSizes.smallBody,
+    padding: theme.spacing(1, 2),
+}));
+
 export const FeatureStrategyMenuCards = ({
     projectId,
     featureId,
     environmentId,
 }: IFeatureStrategyMenuCardsProps) => {
     const { strategies } = useStrategies();
+    const { uiConfig } = useUiConfig();
+
+    const preDefinedStrategies = useMemo(() => {
+        return strategies.filter(
+            strategy => !strategy.deprecated && !strategy.editable
+        );
+    }, [strategies]);
```

Comment:
do we need those memos? also using objects in hooks as references never works correctly unless you have the exact same memory instance. It's comparison by === not by the actual values

Reply:
Yes we can. Removed